### DISCUSSION TITLE
Fields dropping in JSON to HL7v2 transformation

### DISCUSSION THREAD

**Discussion by trlst15** [nextgenhealthcare/connect#4979] [General]
Seeing odd behavior in a transformation I'm performing. The channel in question is reading from an SQS queue and turning the JSON messages into HL7v2.

Most fields are passing successfully. I am, however, seeing that some JSON fields aren't making it into the HL7 message and are instead adding "undefined" to these fields. The missing fields are always present in the JSON and the issue is fixed when I replay the message without modification. Has anyone come across something similar before? 

Thank you.

**Comment by rjkroll** [comment]
You'd have to post some code or your channel.   Its too hard to tell what's going on without knowing what you are doing.
